### ISSUE TITLE
`conda install nb_conda` fails on default Anaconda3-2020.07-Linux-x86_64 install.

### ISSUE THREAD

**Issue by BP-Santo** [conda/conda#10145]
## Current Behavior
`conda install nb_conda` fails as per provided output on a fresh Anaconda3-2020.07-Linux-x86_64 install.

### Steps to Reproduce
```
(base) anaconda@anaconda:~$ conda install nb_conda
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                       

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - nb_conda -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']

Your python: python=3.8

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```

## Expected Behavior
To be able to install the jupyter extensions per documentation [https://docs.anaconda.com/anaconda/user-guide/tasks/use-jupyter-notebook-extensions/](url) with `conda install conda_nb`.

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
(base) anaconda@anaconda:~$ conda info

     active environment : base
    active env location : /home/anaconda/anaconda3
            shell level : 1
       user config file : /home/anaconda/.condarc
 populated config files : 
          conda version : 4.8.3
    conda-build version : 3.18.11
         python version : 3.8.3.final.0
       virtual packages : __glibc=2.28
       base environment : /home/anaconda/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/anaconda/anaconda3/pkgs
                          /home/anaconda/.conda/pkgs
       envs directories : /home/anaconda/anaconda3/envs
                          /home/anaconda/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.3 requests/2.24.0 CPython/3.8.3 Linux/4.19.0-10-amd64 debian/10 glibc/2.28
                UID:GID : 1001:1001
             netrc file : None
           offline mode : False
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```

```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
(base) anaconda@anaconda:~$ conda list --show-channel-urls
# packages in environment at /home/anaconda/anaconda3:
#
# Name                    Version                   Build  Channel
_ipyw_jlab_nb_ext_conf    0.1.0                    py38_0    defaults
_libgcc_mutex             0.1                        main    defaults
alabaster                 0.7.12                     py_0    defaults
anaconda                  2020.07                  py38_0    defaults
anaconda-client           1.7.2                    py38_0    defaults
anaconda-navigator        1.9.12                   py38_0    defaults
anaconda-project          0.8.4                      py_0    defaults
argh                      0.26.2                   py38_0    defaults
asn1crypto                1.3.0                    py38_0    defaults
astroid                   2.4.2                    py38_0    defaults
astropy                   4.0.1.post1      py38h7b6447c_1    defaults
atomicwrites              1.4.0                      py_0    defaults
attrs                     19.3.0                     py_0    defaults
autopep8                  1.5.3                      py_0    defaults
babel                     2.8.0                      py_0    defaults
backcall                  0.2.0                      py_0    defaults
backports                 1.0                        py_2    defaults
backports.functools_lru_cache 1.6.1                      py_0    defaults
backports.shutil_get_terminal_size 1.0.0                    py38_2    defaults
backports.tempfile        1.0                        py_1    defaults
backports.weakref         1.0.post1                  py_1    defaults
beautifulsoup4            4.9.1                    py38_0    defaults
bitarray                  1.4.0            py38h7b6447c_0    defaults
bkcharts                  0.2                      py38_0    defaults
blas                      1.0                         mkl    defaults
bleach                    3.1.5                      py_0    defaults
blosc                     1.19.0               hd408876_0    defaults
bokeh                     2.1.1                    py38_0    defaults
boto                      2.49.0                   py38_0    defaults
bottleneck                1.3.2            py38heb32a55_1    defaults
brotlipy                  0.7.0           py38h7b6447c_1000    defaults
bzip2                     1.0.8                h7b6447c_0    defaults
ca-certificates           2020.6.24                     0    defaults
cairo                     1.14.12              h8948797_3    defaults
certifi                   2020.6.20                py38_0    defaults
cffi                      1.14.0           py38he30daa8_1    defaults
chardet                   3.0.4                 py38_1003    defaults
click                     7.1.2                      py_0    defaults
cloudpickle               1.5.0                      py_0    defaults
clyent                    1.2.2                    py38_1    defaults
colorama                  0.4.3                      py_0    defaults
conda                     4.8.3                    py38_0    defaults
conda-build               3.18.11                  py38_0    defaults
conda-env                 2.6.0                         1    defaults
conda-package-handling    1.6.1            py38h7b6447c_0    defaults
conda-verify              3.4.2                      py_1    defaults
contextlib2               0.6.0.post1                py_0    defaults
cryptography              2.9.2            py38h1ba5d50_0    defaults
curl                      7.71.1               hbc83047_1    defaults
cycler                    0.10.0                   py38_0    defaults
cython                    0.29.21          py38he6710b0_0    defaults
cytoolz                   0.10.1           py38h7b6447c_0    defaults
dask                      2.20.0                     py_0    defaults
dask-core                 2.20.0                     py_0    defaults
dbus                      1.13.16              hb2f20db_0    defaults
decorator                 4.4.2                      py_0    defaults
defusedxml                0.6.0                      py_0    defaults
diff-match-patch          20200713                   py_0    defaults
distributed               2.20.0                   py38_0    defaults
docutils                  0.16                     py38_1    defaults
entrypoints               0.3                      py38_0    defaults
et_xmlfile                1.0.1                   py_1001    defaults
expat                     2.2.9                he6710b0_2    defaults
fastcache                 1.1.0            py38h7b6447c_0    defaults
filelock                  3.0.12                     py_0    defaults
flake8                    3.8.3                      py_0    defaults
flask                     1.1.2                      py_0    defaults
fontconfig                2.13.0               h9420a91_0    defaults
freetype                  2.10.2               h5ab3b9f_0    defaults
fribidi                   1.0.9                h7b6447c_0    defaults
fsspec                    0.7.4                      py_0    defaults
future                    0.18.2                   py38_1    defaults
get_terminal_size         1.0.0                haa9412d_0    defaults
gevent                    20.6.2           py38h7b6447c_0    defaults
glib                      2.65.0               h3eb4bd4_0    defaults
glob2                     0.7                        py_0    defaults
gmp                       6.1.2                h6c8ec71_1    defaults
gmpy2                     2.0.8            py38hd5f6e3b_3    defaults
graphite2                 1.3.14               h23475e2_0    defaults
greenlet                  0.4.16           py38h7b6447c_0    defaults
gst-plugins-base          1.14.0               hbbd80ab_1    defaults
gstreamer                 1.14.0               hb31296c_0    defaults
h5py                      2.10.0           py38h7918eee_0    defaults
harfbuzz                  2.4.0                hca77d97_1    defaults
hdf5                      1.10.4               hb1b8bf9_0    defaults
heapdict                  1.0.1                      py_0    defaults
html5lib                  1.1                        py_0    defaults
icu                       58.2                 he6710b0_3    defaults
idna                      2.10                       py_0    defaults
imageio                   2.9.0                      py_0    defaults
imagesize                 1.2.0                      py_0    defaults
importlib-metadata        1.7.0                    py38_0    defaults
importlib_metadata        1.7.0                         0    defaults
intel-openmp              2020.1                      217    defaults
intervaltree              3.0.2                      py_1    defaults
ipykernel                 5.3.2            py38h5ca1d4c_0    defaults
ipython                   7.16.1           py38h5ca1d4c_0    defaults
ipython_genutils          0.2.0                    py38_0    defaults
ipywidgets                7.5.1                      py_0    defaults
isort                     4.3.21                   py38_0    defaults
itsdangerous              1.1.0                      py_0    defaults
jbig                      2.1                  hdba287a_0    defaults
jdcal                     1.4.1                      py_0    defaults
jedi                      0.17.1                   py38_0    defaults
jeepney                   0.4.3                      py_0    defaults
jinja2                    2.11.2                     py_0    defaults
joblib                    0.16.0                     py_0    defaults
jpeg                      9b                   h024ee3a_2    defaults
json5                     0.9.5                      py_0    defaults
jsonschema                3.2.0                    py38_0    defaults
jupyter                   1.0.0                    py38_7    defaults
jupyter_client            6.1.6                      py_0    defaults
jupyter_console           6.1.0                      py_0    defaults
jupyter_core              4.6.3                    py38_0    defaults
jupyterlab                2.1.5                      py_0    defaults
jupyterlab_server         1.2.0                      py_0    defaults
keyring                   21.2.1                   py38_0    defaults
kiwisolver                1.2.0            py38hfd86e86_0    defaults
krb5                      1.18.2               h173b8e3_0    defaults
lazy-object-proxy         1.4.3            py38h7b6447c_0    defaults
lcms2                     2.11                 h396b838_0    defaults
ld_impl_linux-64          2.33.1               h53a641e_7    defaults
libarchive                3.4.2                h62408e4_0    defaults
libcurl                   7.71.1               h20c2e04_1    defaults
libedit                   3.1.20191231         h14c3975_1    defaults
libffi                    3.3                  he6710b0_2    defaults
libgcc-ng                 9.1.0                hdf63c60_0    defaults
libgfortran-ng            7.3.0                hdf63c60_0    defaults
liblief                   0.10.1               he6710b0_0    defaults
libllvm9                  9.0.1                h4a3c616_1    defaults
libpng                    1.6.37               hbc83047_0    defaults
libsodium                 1.0.18               h7b6447c_0    defaults
libspatialindex           1.9.3                he6710b0_0    defaults
libssh2                   1.9.0                h1ba5d50_1    defaults
libstdcxx-ng              9.1.0                hdf63c60_0    defaults
libtiff                   4.1.0                h2733197_1    defaults
libtool                   2.4.6                h7b6447c_5    defaults
libuuid                   1.0.3                h1bed415_2    defaults
libxcb                    1.14                 h7b6447c_0    defaults
libxml2                   2.9.10               he19cac6_1    defaults
libxslt                   1.1.34               hc22bd24_0    defaults
llvmlite                  0.33.0           py38hc6ec683_1    defaults
locket                    0.2.0                    py38_1    defaults
lxml                      4.5.2            py38hefd8a0e_0    defaults
lz4-c                     1.9.2                he6710b0_0    defaults
lzo                       2.10                 h7b6447c_2    defaults
markupsafe                1.1.1            py38h7b6447c_0    defaults
matplotlib                3.2.2                         0    defaults
matplotlib-base           3.2.2            py38hef1b27d_0    defaults
mccabe                    0.6.1                    py38_1    defaults
mistune                   0.8.4           py38h7b6447c_1000    defaults
mkl                       2020.1                      217    defaults
mkl-service               2.3.0            py38he904b0f_0    defaults
mkl_fft                   1.1.0            py38h23d657b_0    defaults
mkl_random                1.1.1            py38h0573a6f_0    defaults
mock                      4.0.2                      py_0    defaults
more-itertools            8.4.0                      py_0    defaults
mpc                       1.1.0                h10f8cd9_1    defaults
mpfr                      4.0.2                hb69a4c5_1    defaults
mpmath                    1.1.0                    py38_0    defaults
msgpack-python            1.0.0            py38hfd86e86_1    defaults
multipledispatch          0.6.0                    py38_0    defaults
navigator-updater         0.2.1                    py38_0    defaults
nbconvert                 5.6.1                    py38_0    defaults
nbformat                  5.0.7                      py_0    defaults
ncurses                   6.2                  he6710b0_1    defaults
networkx                  2.4                        py_1    defaults
nltk                      3.5                        py_0    defaults
nose                      1.3.7                    py38_2    defaults
notebook                  6.0.3                    py38_0    defaults
numba                     0.50.1           py38h0573a6f_1    defaults
numexpr                   2.7.1            py38h423224d_0    defaults
numpy                     1.18.5           py38ha1c710e_0    defaults
numpy-base                1.18.5           py38hde5b4d6_0    defaults
numpydoc                  1.1.0                      py_0    defaults
olefile                   0.46                       py_0    defaults
openpyxl                  3.0.4                      py_0    defaults
openssl                   1.1.1g               h7b6447c_0    defaults
packaging                 20.4                       py_0    defaults
pandas                    1.0.5            py38h0573a6f_0    defaults
pandoc                    2.10                          0    defaults
pandocfilters             1.4.2                    py38_1    defaults
pango                     1.45.3               hd140c19_0    defaults
parso                     0.7.0                      py_0    defaults
partd                     1.1.0                      py_0    defaults
patchelf                  0.11                 he6710b0_0    defaults
path                      13.1.0                   py38_0    defaults
path.py                   12.4.0                        0    defaults
pathlib2                  2.3.5                    py38_0    defaults
pathtools                 0.1.2                      py_1    defaults
patsy                     0.5.1                    py38_0    defaults
pcre                      8.44                 he6710b0_0    defaults
pep8                      1.7.1                    py38_0    defaults
pexpect                   4.8.0                    py38_0    defaults
pickleshare               0.7.5                 py38_1000    defaults
pillow                    7.2.0            py38hb39fc2d_0    defaults
pip                       20.1.1                   py38_1    defaults
pixman                    0.40.0               h7b6447c_0    defaults
pkginfo                   1.5.0.1                  py38_0    defaults
pluggy                    0.13.1                   py38_0    defaults
ply                       3.11                     py38_0    defaults
prometheus_client         0.8.0                      py_0    defaults
prompt-toolkit            3.0.5                      py_0    defaults
prompt_toolkit            3.0.5                         0    defaults
psutil                    5.7.0            py38h7b6447c_0    defaults
ptyprocess                0.6.0                    py38_0    defaults
py                        1.9.0                      py_0    defaults
py-lief                   0.10.1           py38h403a769_0    defaults
pycodestyle               2.6.0                      py_0    defaults
pycosat                   0.6.3            py38h7b6447c_1    defaults
pycparser                 2.20                       py_2    defaults
pycurl                    7.43.0.5         py38h1ba5d50_0    defaults
pydocstyle                5.0.2                      py_0    defaults
pyflakes                  2.2.0                      py_0    defaults
pygments                  2.6.1                      py_0    defaults
pylint                    2.5.3                    py38_0    defaults
pyodbc                    4.0.30           py38he6710b0_0    defaults
pyopenssl                 19.1.0                     py_1    defaults
pyparsing                 2.4.7                      py_0    defaults
pyqt                      5.9.2            py38h05f1152_4    defaults
pyrsistent                0.16.0           py38h7b6447c_0    defaults
pysocks                   1.7.1                    py38_0    defaults
pytables                  3.6.1            py38h9fd0a39_0    defaults
pytest                    5.4.3                    py38_0    defaults
python                    3.8.3                hcff3b4d_2    defaults
python-dateutil           2.8.1                      py_0    defaults
python-jsonrpc-server     0.3.4                      py_1    defaults
python-language-server    0.34.1                   py38_0    defaults
python-libarchive-c       2.9                        py_0    defaults
pytz                      2020.1                     py_0    defaults
pywavelets                1.1.1            py38h7b6447c_0    defaults
pyxdg                     0.26                       py_0    defaults
pyyaml                    5.3.1            py38h7b6447c_1    defaults
pyzmq                     19.0.1           py38he6710b0_1    defaults
qdarkstyle                2.8.1                      py_0    defaults
qt                        5.9.7                h5867ecd_1    defaults
qtawesome                 0.7.2                      py_0    defaults
qtconsole                 4.7.5                      py_0    defaults
qtpy                      1.9.0                      py_0    defaults
readline                  8.0                  h7b6447c_0    defaults
regex                     2020.6.8         py38h7b6447c_0    defaults
requests                  2.24.0                     py_0    defaults
ripgrep                   11.0.2               he32d670_0    defaults
rope                      0.17.0                     py_0    defaults
rtree                     0.9.4                    py38_1    defaults
ruamel_yaml               0.15.87          py38h7b6447c_1    defaults
scikit-image              0.16.2           py38h0573a6f_0    defaults
scikit-learn              0.23.1           py38h423224d_0    defaults
scipy                     1.5.0            py38h0b6359f_0    defaults
seaborn                   0.10.1                     py_0    defaults
secretstorage             3.1.2                    py38_0    defaults
send2trash                1.5.0                    py38_0    defaults
setuptools                49.2.0                   py38_0    defaults
simplegeneric             0.8.1                    py38_2    defaults
singledispatch            3.4.0.3                  py38_0    defaults
sip                       4.19.13          py38he6710b0_0    defaults
six                       1.15.0                     py_0    defaults
snappy                    1.1.8                he6710b0_0    defaults
snowballstemmer           2.0.0                      py_0    defaults
sortedcollections         1.2.1                      py_0    defaults
sortedcontainers          2.2.2                      py_0    defaults
soupsieve                 2.0.1                      py_0    defaults
sphinx                    3.1.2                      py_0    defaults
sphinxcontrib             1.0                      py38_1    defaults
sphinxcontrib-applehelp   1.0.2                      py_0    defaults
sphinxcontrib-devhelp     1.0.2                      py_0    defaults
sphinxcontrib-htmlhelp    1.0.3                      py_0    defaults
sphinxcontrib-jsmath      1.0.1                      py_0    defaults
sphinxcontrib-qthelp      1.0.3                      py_0    defaults
sphinxcontrib-serializinghtml 1.1.4                      py_0    defaults
sphinxcontrib-websupport  1.2.3                      py_0    defaults
spyder                    4.1.4                    py38_0    defaults
spyder-kernels            1.9.2                    py38_0    defaults
sqlalchemy                1.3.18           py38h7b6447c_0    defaults
sqlite                    3.32.3               h62c20be_0    defaults
statsmodels               0.11.1           py38h7b6447c_0    defaults
sympy                     1.6.1                    py38_0    defaults
tbb                       2020.0               hfd86e86_0    defaults
tblib                     1.6.0                      py_0    defaults
terminado                 0.8.3                    py38_0    defaults
testpath                  0.4.4                      py_0    defaults
threadpoolctl             2.1.0              pyh5ca1d4c_0    defaults
tk                        8.6.10               hbc83047_0    defaults
toml                      0.10.1                     py_0    defaults
toolz                     0.10.0                     py_0    defaults
tornado                   6.0.4            py38h7b6447c_1    defaults
tqdm                      4.47.0                     py_0    defaults
traitlets                 4.3.3                    py38_0    defaults
typing_extensions         3.7.4.2                    py_0    defaults
ujson                     1.35             py38h7b6447c_0    defaults
unicodecsv                0.14.1                   py38_0    defaults
unixodbc                  2.3.7                h14c3975_0    defaults
urllib3                   1.25.9                     py_0    defaults
watchdog                  0.10.3                   py38_0    defaults
wcwidth                   0.2.5                      py_0    defaults
webencodings              0.5.1                    py38_1    defaults
werkzeug                  1.0.1                      py_0    defaults
wheel                     0.34.2                   py38_0    defaults
widgetsnbextension        3.5.1                    py38_0    defaults
wrapt                     1.11.2           py38h7b6447c_0    defaults
wurlitzer                 2.0.1                    py38_0    defaults
xlrd                      1.2.0                      py_0    defaults
xlsxwriter                1.2.9                      py_0    defaults
xlwt                      1.3.0                    py38_0    defaults
xmltodict                 0.12.0                     py_0    defaults
xz                        5.2.5                h7b6447c_0    defaults
yaml                      0.2.5                h7b6447c_0    defaults
yapf                      0.30.0                     py_0    defaults
zeromq                    4.3.2                he6710b0_2    defaults
zict                      2.0.0                      py_0    defaults
zipp                      3.1.0                      py_0    defaults
zlib                      1.2.11               h7b6447c_3    defaults
zope                      1.0                      py38_1    defaults
zope.event                4.4                      py38_0    defaults
zope.interface            4.7.1            py38h7b6447c_0    defaults
zstd                      1.4.5                h0b5b093_0    defaults

```
</p></details>

**Comment by chenghlee** [comment]
There is currently no Python 3.8 build for `nb_conda` in the "defaults" channel. Give us a few hours, and we'll have one up.

**Comment by chenghlee** [comment]
@BP-Santo: The `nb_conda` package should now be available for Python 3.8; please try installing it, and if it works, feel free to close this ticket.

**Comment by BP-Santo** [comment]
Thank you very much for the quick response. I managed to pull nb_conda fine however I stumbled into this issue [https://github.com/Anaconda-Platform/nb_conda/issues/66](url).

I managed to find a forked project [https://github.com/fcollonval/jupyter_conda](url)
Would it be better to just use that rather than try and apply community suggested fixed to nb_conda?
